src/frontend/context/OppgaveContext.ts

```diff
@@ -12,14 +14,21 @@ import {
     RessursStatus,
     RessursSuksess,
 } from '../typer/ressurs';
+import { harStrengtFortroligRolle } from '../utils/roller';
 
 export const [OppgaveProvider, useOppgave] = constate(() => {
-    const { request } = useApp();
+    const { request, saksbehandler, appEnv } = useApp();
     const [oppgaveRessurs, settOppgaveRessurs] =
         useState<Ressurs<OppgaverResponse>>(byggTomRessurs());
     const [laster, settLaster] = useState<boolean>(false);
+    const [lasterOppgaveRequestFraLokalt, settLasterOppgaveRequestFraLokalt] =
+        useState<boolean>(false);
```

Comment:
Brukes denne? Ser ikke ut som den settes til true på noe tidspunkt

Reply:
Yep, den hette tidligere `lasterFraLokalt`. Den er flyttet for at hentingen gjøres i contexten
https://github.com/navikt/tilleggsstonader-sak-frontend/pull/340/commits/ec3a82b25314b482365c494efc10e21409bb60cf#diff-fcd2151872c640350ecdb08f4bc9efc1d9e318719c9e482c0df983a1cda2b472L55

Reply:
Men nå har den default `false` og settes aldri til `true`? 🤔

Reply:
Ble feil i flyttingen av den, fikset i https://github.com/navikt/tilleggsstonader-sak-frontend/pull/340/commits/7e0ca61e4a7d19eb8f6f979b1611cec3eeeeb48b